index.bs

```diff
@@ -12521,6 +12518,20 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="isextensible">Platform object \[[IsExtensible]] method</h4>
+
+The internal \[[IsExtensible]] method of every [=platform object=] |O|
+must behave as follows:
+
+<ol class="algorithm">
+    1.  If |O| implements an [=interface=] that
+        [=support indexed properties|supports indexed properties=] or
+        [=support named properties|supports named properties=],
```

Comment:
It looks like other internal methods just say "If O supports indexed properties", e.g. https://heycam.github.io/webidl/#defineownproperty. A bit surprising; I think your phrasing is more correct. But probably best to stay consistent for now, and I'll file a follow-up editorial issue.